src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -2145,6 +2166,15 @@ private void UpdateShownPatchDetails()
 
         // Enable move to patch button if this is a valid move
         moveToPatchButton.Disabled = !editor.IsPatchMoveValid(patch);
+
+        reportTabPatchSelector.Select(reportTabPatchSelector.GetItemIndex(patch.ID));
+    }
+
+    private void OnReportTabPatchListSelected(int index)
+    {
+        var patch = editor.CurrentGame.GameWorld.Map.Patches[index];
```

Comment:
This is not correct, the patches find order shouldn't be assumed to be the same.

This should instead get the item ID based on the index from the patch selector, and then from the map get the patch by the ID.

Reply:
Fixed.